registrations/models.py

```diff
@@ -237,6 +238,47 @@ class WebStoreMerchant(CreatedModifiedBaseModel):
         default="",
     )
 
+    def to_web_store_merchant_json(self):
+        return {
+            "merchantName": self.name,
+            "merchantStreet": self.street_address,
+            "merchantZip": self.zipcode,
+            "merchantCity": self.city,
+            "merchantEmail": self.email,
+            "merchantPhone": self.phone_number,
+            "merchantUrl": self.url,
+            "merchantTermsOfServiceUrl": self.terms_of_service_url,
+            "merchantBusinessId": self.business_id,
+            "merchantPaytrailMerchantId": self.paytrail_merchant_id,
+        }
+
+    def delete(self, using=None, keep_parents=False, force_delete=False):
+        if force_delete:
+            super().delete()
```

Comment:
We could benefit logging the deletion (in case of future debugging weirdness) 🗒️

Reply:
True, this can turn out helpful. Added logging through `logger.info`.